packages/wallet/.eslintrc.js

```diff
@@ -1,49 +1,89 @@
+const generalRules = {
+  "no-var": "off",
```

Comment:
Why would we want this?
`let` and `const` should be used no?
Also, I am also a strong believer in using: `"prefer-const": "error",` as well.

Reply:
This is not a rule I am enforcing on anyone, it is what was already used in the codebase.